remediation_worker/jobs/rds_remove_public_endpoint/README.md

```diff
@@ -0,0 +1,74 @@
+# Private rds snapshot
+
+This job disables rds public accessible to ensure it is not connected by any internet hosts.
```

Comment:
This job removes public access from RDS instances and makes the instances private.